Core/Core/Login/LoginSession.swift

```diff
@@ -146,6 +146,47 @@ public struct LoginSession: Codable, Hashable {
         )
     }
 
+    // MARK: - Migrate Previously-saved annotated PDF documents
+
+    public func migrateSavedAnnotatedPDFs() {
+        let fileManager = FileManager.default
+
+        do {
+            // Make sure `AnnotatedPDFs` folder exists.
+            try fileManager.createDirectory(at: URL.Directories.annotatedPDFs, withIntermediateDirectories: true)
+
+            // Fetch `Documents` shallow contents
+            let urls = try fileManager.contentsOfDirectory(at: URL.Directories.documents, includingPropertiesForKeys: nil)
+
+            guard let folderUrl = urls.first(where: { $0.hasDirectoryPath && $0.lastPathComponent == uniqueID })
+            else { return }
+
+            let sessionFolder = URL
+                .Directories
+                .annotatedPDFs
+                .appending(component: folderUrl.lastPathComponent, directoryHint: .isDirectory)
+
+            if fileManager.fileExists(atPath: sessionFolder.path()) == false {
+                try fileManager.createDirectory(at: sessionFolder, withIntermediateDirectories: true)
+            }
+
+            try? fileManager
+                .contentsOfDirectory(at: folderUrl, includingPropertiesForKeys: nil)
+                .filter({ $0.lastPathComponent.hasPrefix(URL.Paths.offline) == false })
```

Comment:
Thanks for the filtering here. My concern with this is that in case we want to save something in the user's session folder in the future it will be copied to the annotated pdf directory since this only excludes the offline folder.
All the annotated pdfs are saved in directories with only numbers in their names and the annotated files have a pdf extension. Could we use this knowledge to move only such directories?